tesseract_core/sdk/config.py

```diff
@@ -62,21 +65,29 @@ def update_config(**kwargs: Any) -> None:
     global _current_config
 
     conf_settings = {}
+
     for field in RuntimeConfig.model_fields.keys():
         env_key = f"TESSERACT_{field.upper()}"
         if env_key in os.environ:
             conf_settings[field] = os.environ[env_key]
 
     conf_settings.update(kwargs)
 
-    config = RuntimeConfig(**conf_settings)
+    try:
+        config = RuntimeConfig(**conf_settings)
+    except PydanticValidationError as err:
+        raise ValidationError(f"Invalid configuration: {err}") from err
+    except (FileNotFoundError, PermissionError) as err:
+        raise ValidationError(f"Executable not found or not executable: {err}") from err
```

Comment:
Why "convert" different exceptions into `ValidationError`s here? Isn't it better to let the exceptions flow upwards and be handled upstream?